applications/API/urls.py

```
@@ -0,0 +1,22 @@
+from applications.API import views
+from applications.API import endpoints
+
+# router = DefaultRouter()
+# router.register(r"insults", views.InsultViewSet, basename="insult")
+# router.register(r"categories", views.CategoryViewSet, basename="category")
+
+urlpatterns = [
+    # API documentation/schema endpoints
+    path("schema/", SpectacularAPIView.as_view(), name="schema"),
+    path("swagger/", SpectacularSwaggerView.as_view(), name="swagger"),
+    path("redoc/", SpectacularRedocView.as_view(), name="redoc")
```

Pylint (3.3.7)

[error] Parsing failed: 'invalid syntax. Perhaps you forgot a comma? (git.applications.API.urls, line 20)'

(E0001)